package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/companieshouse/officer-filing-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.115",
+    "@companieshouse/api-sdk-node": "^2.0.117",
```

Comment:
being selfish here but if we can update to the latest 119, I wont have to fix conflict in my PR :p